tools/DfmHttpService/Handler/ExitHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DfmHttpService
+{
+    using System.Threading.Tasks;
+
+    public class ExitHandler : IHttpHandler
+    {
+        public bool IsSupport(ServiceContext context)
+        {
+            return context.Message.Name == CommandName.Exit;
+        }
+
+        public Task HandleAsync(ServiceContext context)
+        {
+            return Task.Run(() =>
+            {
+                Utility.ReplyExitResponse(context.HttpContext, "Dfm service exit");
```

Comment:
fixed in latest commit

---
In reply to: [95717861](https://github.com/dotnet/docfx/pull/1145#discussion_r95717861) [](ancestors = 95717861,95717241,95716842)